src/backend/demo/management/commands/create_demo.py

```diff
@@ -117,19 +117,20 @@ def create_demo(stdout):
         last_names = [fake.last_name() for _ in range(name_size)]
         for i in range(defaults.NB_OBJECTS["users"]):
             first_name = random.choice(first_names)
-            queue.push(
-                models.User(
-                    admin_email=f"user{i:d}@example.com",
-                    email=f"user{i:d}@example.com",
-                    password="!",
-                    is_superuser=False,
-                    is_active=True,
-                    is_staff=False,
-                    short_name=first_name,
-                    full_name=f"{first_name:s} {random.choice(last_names):s}",
-                    language=random.choice(settings.LANGUAGES)[0],
+            if not models.User.objects.filter(admin_email=f"user{i:d}@example.com").exists():
```

Comment:
Can you explain how the admin email can be duplicate since it is computed with the index of the loop. My understanding is that it is necessary unique per user :thinking: